corehq/apps/app_manager/models.py

```diff
@@ -4114,7 +4114,7 @@ def wrap(cls, data):
         if "_attachments" in data:
             data = data.copy()
             attachments = data.pop("_attachments").copy()
-            if cls.migrating_blobs_from_couch:
+            if cls._migrating_blobs_from_couch:
```

Comment:
Previously this had always evaluated to `True` because jsonobject replaced the class attribute boolean value with a `jsonobject.properties.BooleanProperty`. It now works as expected, which is why `_attachments` had to be removed from some test JSON docs.